README.Rmd

```diff
@@ -64,7 +64,7 @@ The newest version of the package can be installed from Github:
 
 ```{r, eval=FALSE}
 library(devtools)
-install_github("michaelweylandt/MoMA", ref = "build")
+install_github("DataSlingers/MoMA", ref = "develop")
```

Comment:
I think we want folks to install from the (built) `master` branch, not `develop.`